redshift_sqlalchemy/dialect.py

```diff
@@ -19,6 +19,12 @@ class RedshiftImpl(postgresql.PostgresqlImpl):
         __dialect__ = 'redshift'
 
 
+class RedshiftCompiler(PGCompiler):
```

Comment:
I think the subclassing was an issue before because of the name being set on the dialect, it won't be an issue here